src/domain/LevelState.ts

```diff
@@ -0,0 +1,12 @@
+import { reactive } from 'vue'
+import type { Level } from './Level'
+
+export interface LevelState {
+  currentLevel: Level | null
+  nextLevel: Level | null
+}
+
+export const levelState: LevelState = reactive({
```

Comment:
The state of the application should not be in the domain folder. It should be in src/stores/...